assets/src/edit-story/components/library/panes/text/textSets/textSets.js

```diff
@@ -83,7 +82,34 @@ function TextSets() {
   );
 
   useEffect(() => {
-    getTextSets().then(setTextSets);
+    window.clearTimeout(loadingTimeoutRef.current);
+    isLoadingRef.current = false;
```

Comment:
I think we can get rid of these if we contain the async stuff to that one effect